src/monitor.c

```diff
@@ -44,7 +44,7 @@ bool al_get_monitor_info(int adapter, ALLEGRO_MONITOR_INFO *info)
       }
    }
 
-   info->x1 = info->y1 = info->x2 = info->y2 = INT_MAX;
+   info->x1 = info->y1 = info->x2 = info->y2 = info->dpi = INT_MAX;
```

Comment:
Can we set info->dpi to 0 before the vt->get_monitor_info call? That way platforms who do not report DPI would report it as 0 instead of leaving it completely undefined.

Reply:
Yes, it makes more sense to me too. Seeing this line here made me think a negative value means an invalid or failed query by convention.

Reply:
Oh, I just understood what you meant, missed that it returns earlier, and it will be garbage. Right, fixing it now, thanks.